app/flatpak-builtins-create-usb.c

```diff
@@ -151,6 +151,7 @@ add_related (GHashTable   *all_refs,
         {
           g_printerr (_("Warning: Omitting related ref ‘%s’ because it is not installed.\n"),
                       ext->ref);
+          g_clear_error (&error);
```

Comment:
One too many indirections here. error is already a GError **

Reply:
Probably also makes more sense just to pass `NULL` rather than `error`